### PR TITLE
Rename "existing rules file" to example rules

### PR DESCRIPTION
**Pull request by e-pettersson-ericsson**
### Applicable Issues
Found some inconsistencies in the documentation about our example rules. Some sentences were hard to read, making it difficult to understand.

### Description of the Change
Renamed existing-rules-files.md to example-rules, and updated the links to this file.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Emelie Pettersson emelie.pettersson@ericsson.com

### CODE REVIEW

File: wiki/markdown/example-rules.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Example rules
+
+We have created some example rules that represent use cases which triggered 
+the development of Eiffel Intelligence. The files containing the rules 
+can be found [**here**](https://github.com/Ericsson/eiffel-intelligence/tree/master/src/main/resources/rules).
```
Comment (tobiasake) [inline wiki/markdown/example-rules.md]: should be eiffel-community in link.

File: wiki/markdown/merge-resolver-rules.md [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+**ExtractionRules**. If **MergeResolverRules** is null, the **ExtractionRules** 
+object will be inserted to the root of aggregated object.
+
+## Specifying location with MergeResolverRules
+The location in the aggregated object where the extracted content will be 
+merged, can be specified with path elements. The path element needs a 
+value to exist, but not all elements of a path are needed. The aggregated 
+object will be flattened and all paths that contains the path elements in 
+the given order will be collected. Then we check the value, and the path 
+with contains the value is the winning path. If no value from the paths
```
Comment (tobiasake) [inline wiki/markdown/merge-resolver-rules.md]: with -> should be "which" ?

Comment (e-pettersson-ericsson) [reply wiki/markdown/merge-resolver-rules.md]: yes, fixed :)

File: wiki/markdown/example-rules.md (hunk 1 above)
Comment (e-pettersson-ericsson) [reply wiki/markdown/example-rules.md]: fixed

File: wiki/markdown/merge-resolver-rules.md [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+the given order will be collected. Then we check the value, and the path 
+which contains the value is the winning path. If no value from the paths 
+match the given value then we take the longest path that contains the 
+given path elements.
+
+If you want to place the extracted content into the root of the aggregated
+object you can choose to specify a location key which does not exist in 
+the aggregated object, and there Eiffel Intelligence will not find this
+location. In our example rules we use the non-existing key and value 
+**NONEPATH:NONE** to make Eiffel Intelligence place the content at the
```
Comment (Christoffer-Cortes) [inline wiki/markdown/merge-resolver-rules.md]: I know you mentioned this was from the example rules but why is this not called PATH:NONE. What's up with the repeated NONE. =S

Comment (e-pettersson-ericsson) [reply wiki/markdown/merge-resolver-rules.md]: Maybe to make it clearer that the path doesn't exist?

**Pull request merged**